extension/src/content/index.js

```diff
@@ -132,33 +131,32 @@ function init() {
   }
 
   function createGithubPrettierButtons() {
-    const BUTTON_STYLE = { float: "left", "margin-right": "10px" };
     const createList = seachForGithubButtons();
 
     for (const button of createList) {
-      let prettierBtn = button.parentNode.querySelector(".prettier-btn");
-      if (prettierBtn === null) {
-        prettierBtn = renderButton(button.parentNode, {
+      if (!button.parentNode.querySelector(".prettier-btn")) {
+        const prettierBtn = renderButton(button.parentNode, {
           append: true,
           classes: ["prettier-btn"],
-          refNode: button,
-          style: BUTTON_STYLE
+          refNode: button.innerText === SUBMIT_NEW_ISSUE ? button : null,
+          style: { "margin-right": "4px" }
         });
-      }
 
-      const textArea = findWithClass(prettierBtn, "comment-form-textarea");
-      prettierBtn.addEventListener("click", event => {
-        event.preventDefault();
-        const formattedText = prettier.format(textArea.value, {
-          parser: "markdown",
-          plugins: prettierPlugins,
-          ...storageCache.options
+        const textArea = findWithClass(prettierBtn, "comment-form-textarea");
+
+        prettierBtn.addEventListener("click", event => {
```

Comment:
This fixes a memory leak we currently have where this would bind this event listener even when the button existed. This can be demonstrated by going to the Conversation view of a PR, editing an existing comment, and clicking the Prettier button repeatedly.

Reply:
This actually isn't the right fix here - it now doesn't bind the Files Changed buttons correctly :( Going to take a different approach.